src/components/ReviewInfoText/index.tsx

```diff
@@ -6,8 +6,15 @@ import Paragraph from 'src/components/layout/Paragraph'
 import { currentSafe } from 'src/logic/safe/store/selectors'
 import { lg, sm } from 'src/theme/variables'
 import { TransactionFees } from '../TransactionsFees'
+import useLastQueuedTxNonce from 'src/routes/safe/components/Transactions/TxList/hooks/useLastQueuedTxNonce'
 
-type ReviewInfoTextProps = Parameters<typeof TransactionFees>[0] & { safeNonce?: string; testId?: string }
+type CustomReviewInfoTextProps = {
+  safeNonce?: string
+  lastTxNonce?: number
```

Comment:
I'm not following here – is it a prop or is it a local var from a selector?

Reply:
Good catch, I forgot to clean here when implemented the hook for it.